marti_common_msgs/msg/HealthStatus.msg

```diff
@@ -0,0 +1,12 @@
+# This message is intended for use as a light-weight machine status indicator for robot consumption
+# whereas Diagnostics are for human consumption
+
+int8 OK=0    # Initialized with no issues
```

Comment:
The rosmsg doc says `byte: deprecated alias for int8`, so might as well just leave this as int8.